code/processes/tickerlogreplay.q

```diff
@@ -216,11 +219,23 @@ replaylog:{[logfile]
 
 // If replay date in HDB, delete tables/partition from the HDB so no data is duplicated
 cleanhdb:{[dt]
-  if[not (`$string dt) in key .replay.hdbdir;:()];
+  if[not (`$sd:string dt) in key .replay.hdbdir;.lg.o[`cleanhdb;"Date ",sd," not in HDB."];:()];
   delpaths:.os.pth each .Q.par[.replay.hdbdir;dt;] each $[`all~first .replay.tablelist;enlist `;.replay.tablestoreplay];
   {.lg.o[`cleanhdb;"Deleting ",x," from HDB."];.os.deldir x} each delpaths;
  };
 
+// Replay log file, if file is zipped and the kdb+ version is at least 4.0 then replay through named pipe
+replayinner:{[msgnum;logfile]
+  if[not .replay.zipped;-11!(msgnum;logfile);:()];
+  if[.z.K<4.0;.lg.e[`replaylog;m:"Zipped log files can only be directly replayed on kdb+ 4.0 or higher"];'m];
```

Comment:
should we also check OS here? I think named pipes are only Linux, or maybe macos as well - pretty sure not Windows anyway.